src/App.js

```diff
@@ -1,24 +1,70 @@
-import logo from './logo.svg';
+
+import { useState } from 'react';
 import './App.css';
+import originalContacts from "./contacts.json"
 
 function App() {
+  const fiveContacts = originalContacts.slice(0, 10)
```

Comment:
We need to be consistent with the name of the variables in this cade should be 5
```suggestion
  const fiveContacts = originalContacts.slice(0, 5)
```